lib/tarfiles.py

```diff
@@ -83,7 +84,7 @@ def dcache_persistent_path(exp: str, filename: str) -> str:
     return res
 
 
-def do_tarballs(args: Dict[str, str]) -> None:
+def do_tarballs(args: argparse.Namespace) -> None:
```

Comment:
Should we be leaking the `argparse` or would it be better to pass along specific arguments?

Reply:
The reason we're passing in `args` here is because so much of the code-base, like previous jobsub, is written to operate on the entire Namespace and pick what it needs.  We can certainly change that, but it'd require a significant rewrite (we're talking at least most of the credentials stuff, condor stuff, etc.).  I'd be happy to revisit that, and perhaps we should, but what are your thoughts on opening a new issue for that?

Reply:
Yeah that's fair. Don't bother with an issue, maybe someone will have a reason refactor that in the future, or maybe not.